guiEditor/src/diagram/workbench.tsx

```diff
@@ -199,9 +192,7 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     }
 
     componentDidMount() {
-        this._rootContainer = this.props.globalState.hostDocument.getElementById("workbench-container") as HTMLDivElement;
-        this._gridCanvas = this.props.globalState.hostDocument.getElementById("workbench-canvas-container") as HTMLDivElement;
-        this._svgCanvas = this.props.globalState.hostDocument.getElementById("workbench-svg-container") as HTMLElement;
+        this._rootContainer = this.props.globalState.hostDocument.getElementById("workbench-canvas") as HTMLDivElement;

```

Comment:
Why not using React Ref here as the element is part of the same component ? (won t change a lot cause there can only be one workbench per page in this case but still sounds like a good practice)